src/models/CourseModel.js

```diff
@@ -0,0 +1,103 @@
+const admin = require('firebase-admin');
+
+class CourseModel {
+  constructor({
+    coursesCode,
+    courseName,
+    typeOfCourse,
+    description,
+    courseImgUrl,
+    logoUrl,
```

Comment:
How is this different than `courseImgUrl`?

Reply:
I will discuss with Edith.